src/components/comic/Header.tsx

```diff
@@ -1,65 +1,85 @@
 import React from 'react'
 import { Text } from '../ui/Text'
 import { Comic } from '@/models/comic'
-import { IconLink } from '../shared/IconLink'
-import WebsiteIcon from 'public/assets/vector-icons/web-icon.svg'
-import TwitterIcon from 'public/assets/vector-icons/twitter-icon.svg'
-import DiscordIcon from 'public/assets/vector-icons/discord-icon.svg'
-import TelegramIcon from 'public/assets/vector-icons/telegram-icon.svg'
-import InstagramIcon from 'public/assets/vector-icons/instagram-icon.svg'
-import TikTokIcon from 'public/assets/vector-icons/tiktok-icon.svg'
-import YouTubeIcon from 'public/assets/vector-icons/youtube-icon.svg'
-import { ComicStats } from './Stats'
-import { FlavorText } from '../shared/FlavorText'
-import { CreatorInfoLink } from '../creator/InfoLink'
-import { GenresList } from '../shared/GenresList'
+import { Socials } from '../shared/Socials'
+import { AvatarImage } from '../shared/AvatarImage'
+import { cn } from '@/lib/utils'
+import { AudienceWidget } from '../shared/AudienceWidget'
+import { StatsList } from '../shared/StatsList'
+import { StatsItem } from '../shared/StatsItem'
+import { RateButton } from '../shared/buttons/RateButton'
+import { FavouritiseButton } from '../shared/buttons/FavouritiseButton'
+import { BookmarkButton } from '../shared/buttons/BookmarkButton'
+import { TextWithViewMoreButton } from '../ui/TextWithViewMoreButton'
 
 type Props = {
   comic: Comic
 }
 
 export const ComicHeader: React.FC<Props> = ({ comic }) => (
-  <div className='flex flex-col -mt-[100px] md:flex-row md:justify-between'>
+  <div className='flex flex-col items-center gap-3 sm:gap-8 sm:flex-row sm:justify-between sm:h-[500px] sm:pb-8 sm:items-end'>
     <LeftSection comic={comic} />
     <RightSection comic={comic} />
   </div>
 )
 
-const LeftSection: React.FC<Props> = ({ comic }) => {
-  const { description, flavorText, genres, title } = comic
-  return (
-    <div className='flex flex-col gap-2 w-full md:max-w-[680px]'>
-      <Text className='font-normal' as='h2' styleVariant='primary-heading'>
-        {title}
-      </Text>
-      {genres?.length ? <GenresList genres={genres} /> : null}
-      <FlavorText text={flavorText} />
-      <Text as='p' className='whitespace-pre-wrap mb-2' styleVariant='body-large'>
-        {description}
-      </Text>
-      <CreatorInfoLink creator={comic.creator} />
+const LeftSection: React.FC<Props> = ({ comic }) => (
+  <div className='flex flex-col gap-3 sm:gap-7 w-full sm:max-w-[500px] lg:max-w-[680px] max-sm:items-start sm:justify-end'>
+    <Socials website={comic.website} instagram={comic.instagram} twitter={comic.twitter} className='sm:hidden' />
+    <Text className='font-normal' as='h3' styleVariant='primary-heading'>
```

Comment:
remove className from this component and it will apply correct styling. Please check the whole text if has the correct font and font weight